src/pages/LoginPage.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div class="login-page">
+    <p class="login-page__hi">Hi there!</p>
+    <p class="login-page__care">Care to log in?</p>
+    <LoginCard />
+    <h5 class="login-page__forgot-password">
+      Did you forget your password? <span class="reset">Reset it now</span>
+    </h5>
```

Comment:
and this does not looks like a heading 😛

Reply:
Feel really silly now :(
THANK YOU

Reply:
that definitely wasn't my intention, please don't!